packages/core/src/components/advanced-select/advanced-select.scss

```diff
@@ -0,0 +1,186 @@
+@use "sass:math";
+@use "~dso-toolkit/src/utilities";
+@use "~dso-toolkit/src/global/mixins/set-colors.mixin" as set-colors;
+@use "~dso-toolkit/src/components/advanced-select";
+
+:host {
+  display: block;
+}
+
+@include utilities.box-sizing();
+
+.active-option {
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+  border: 1px solid advanced-select.$border-color;
+  border-radius: advanced-select.$border-radius;
+  cursor: pointer;
+  background-color: advanced-select.$bg-color;
+  line-height: inherit;
+  font-family: inherit;
+  font-size: inherit;
```

Comment:
de 3 regels hierboven: `line-height`, `font-family` en `font-size` zijn out-of-the-box al inheriting properties, dus vanwaar dit? werkt het niet zonder?

Reply:
Totzover ik begrijp hebben form-elementen geen inheritance van font-properties. Vandaar dat ik dit heb gedaan. Als ik het niet doe ziet het er zo uit: 
![image](https://github.com/dso-toolkit/dso-toolkit/assets/6705457/fb648167-d792-48cd-b6a8-734d51fd3450)

Reply:
Ah, ik had me niet gerealiseerd dat het om een form-element ging. Keek puur naar de css. My bad.